tests/features/dataset.py

```diff
@@ -46,10 +46,14 @@ def is_dataset_exists(self, dataset_id):
             bool: True if the dataset exists, False otherwise.
         """
         try:
-            self.get_dataset_by_id(dataset_id)
-            return True
+            dataset = self.get_dataset_by_id(dataset_id)
+            dataset_data = dataset.data
+            if dataset_data is not None and len(dataset_data) > 0:
```

Comment:
remove if else and return directly, both here and in project.py.

Reply:
Sure, I can do that. Let me update the PR with this change.

Reply:
Great!